blackbox/docs/configuration.txt

```diff
@@ -648,9 +648,22 @@ All current applied cluster settings can be read by querying the
 :ref:`sys.cluster.settings <sys-cluster-settings>` column. Most
 cluster settings can be changed at runtime using the
 :ref:`SET/RESET<administration-set-reset>` statement. This is documented at each setting.
-It's not recommended to add cluster wide settings to the crate.yml file of each node,
-as this will result to each node having a different setting which can lead to
-a non-deterministic behavior of the cluster.
+
+
+.. _updating-cluster-settings:
+
+Updating Cluster Settings
+-------------------------
+
```

Comment:
my proposal:

Applying Non-Runtime Cluster Wide Settings
------------------------------------------

Cluster wide settings which cannot be changed at runtime need to be
specified in the configuration of each node in the cluster.

.. NOTE::

   Cluster settings specified via node configurations are required to
   be exactly the same on every node in the cluster for propper
   operation of the cluster.

Such settings can be changed on a running cluster by first rolling out
the new configuration to all nodes in the cluster and then restart one
of the cluster nodes to apply the new configuration.